common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -48,7 +48,7 @@ public class Configuration {
     private final SharedPreferences prefs;
     private final Resources res;
 
-    public static final String PREFS_KEY_BTC_PRECISION = "btc_precision";
+    public static final String PREFS_KEY_BTC_PRECISION = "btc_precision"; // TODO: this is never changed. Do we need it as a preference?
```

Comment:
This is a leftover from Bitcoin Wallet that allows the user to specify the "precision" or rather the units of the cryptocurrency:

BTC / DASH
mBTC / mDASH (milli)
uBTC / uDASH (micro)

We may still have some PNG files that for these older units of currency.